src/pattern-library/components/patterns/prototype/SelectNextPage.tsx

```diff
@@ -425,6 +425,39 @@ export default function SelectNextPage() {
               withSource
             />
           </Library.Example>
+          <Library.Example title="multiple">
+            <Library.Info>
+              <Library.InfoItem label="description">
+                Determines if more than one item can be selected at once,
+                causing the listbox to stay open when an option is selected on
+                it.
+                <p>Considerations:</p>
```

Comment:
I think this paragraph and list can be simplified to just a description of how other aspects of the API change: "When multi-selection is enabled, `value` must be an array and `onChange` will receive an array as an argument". I think it is easier to follow if you only mention what changes and not aspects (eg. the fact that the caller maintains the state) which remain unchanged.